### PR TITLE
Added coverage support to jest preset

### PR DESCRIPTION
**Pull request by lukaszgolder**

### CODE REVIEW

**Comment by lukaszgolder** [comment]
@eliperelman Done

File: docs/presets/neutrino-preset-jest/README.md [hunk 1]
```diff
@@ -144,6 +145,26 @@ Use the command line [`files` parameters](/cli/README.md#neutrino-test) to execu
...
 
+## Coverage reporting
+
+Jest has an integrated coverage reporter, which requires no configuration. To collect test coverage information and
+generate report run:
+
+#### Yarn
+
+```bash
+❯ yarn test -- --coverage
```
Comment (eliperelman) [inline docs/presets/neutrino-preset-jest/README.md]: This assumes that they are running the command via a package script, which may not always be true. Let's change these particular scripts to:

```bash
❯ neutrino test --coverage
```

Then maybe mention something about a script target like:

```json
{
  "scripts": {
    "coverage": "neutrino test --coverage"
  }
}
```

Thoughts?

File: docs/cli/README.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ Options:
   --presets  A list of Neutrino presets used to configure the build    [array] [default: []]
   --version  Show version number                                       [boolean]
   --help     Show help                                                 [boolean]
+  --coverage Collect test coverage information and generate report     [boolean] [default: false]
   --watch    Watch source files for changes and re-run tests           [boolean] [default: false]
 ```
```
Comment (eliperelman) [inline docs/cli/README.md]: Can you add a description of the `--coverage` flag at around line 95? The `--watch` flag has a similar description right above this.

**Comment by eliperelman** [comment]
Tested this locally, works great! Published this to npm for `neutrino` and `neutrino-preset-jest` at v4.3.0.

**Pull request merged**